coral-component-autocomplete/src/tests/test.Autocomplete.js

```diff
@@ -1945,7 +1945,9 @@ describe('Autocomplete', function() {
 
           el.on('coral-overlay:close', function() {
             helpers.next(function() {
-              expect(trigger.getAttribute('aria-expanded')).to.equal('false');
+              if (trigger) {
```

Comment:
You can remove this in favor of https://github.com/adobe/coral-spectrum/commit/1c38579cdd9b1d941be35313c430c0c20f3f5aad